ehcache-xml/build.gradle

```diff
@@ -115,6 +115,7 @@ dependencies {
 
   xjcToolJakarta 'com.sun.xml.bind:jaxb-xjc:3.0.0-M4'
   xjcToolJakarta 'com.sun.xml.bind:jaxb-impl:3.0.0-M4'
+  xjcToolJakarta 'javax.xml.bind:jaxb-api:[2.2,3)'
```

Comment:
Needed to add the dependency for compilation but I am not sure this is ok (regarding scopes)